src/node.cc

```diff
@@ -4611,15 +4640,18 @@ inline int Start(uv_loop_t* event_loop,
     Locker locker(isolate);
     Isolate::Scope isolate_scope(isolate);
     HandleScope handle_scope(isolate);
-    IsolateData isolate_data(
+    std::unique_ptr<IsolateData, decltype(&FreeIsolateData)> isolate_data(
+      CreateIsolateData(
         isolate,
         event_loop,
         v8_platform.Platform(),
-        allocator.zero_fill_field());
+        allocator.get()),
+      &FreeIsolateData);
```

Comment:
Tiniest of nits but can you indent by four spaces (and the arguments to CreateIsolateData() by eight)?  Helps it stand out as a line continuation.

One glorious day when we switch to clang-format that will all be automatic.